Run on 2022-01-26 08:36:41.531165
Run on 2022-01-26 08:37:59.137298
Run on 2022-01-26 08:39:33.099695
Run on 2022-01-26 08:46:44.364717
Run on 2022-01-26 08:47:55.668314
Run on 2022-01-26 09:02:55.170103
Run on 2022-01-26 09:03:18.642521
Run on 2022-01-26 10:10:04.965516
Run on 2022-01-26 10:13:23.505445
exception occured in name lseu-0.100000 submode remark_2 algorithm MT
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 16, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,False,False,False,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein45-0.900000 submode remark_2 algorithm MT
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 16, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,False,False,False,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein27-0.900000 submode remark_2 algorithm MT
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 16, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,False,False,False,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein45-0.500000 submode remark_2 algorithm MT
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 16, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,False,False,False,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

Run on 2022-01-26 10:20:24.669004
exception occured in name lseu-0.100000 submode remark_2 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein45-0.900000 submode remark_2 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein27-0.900000 submode remark_2 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein45-0.500000 submode remark_2 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

Run on 2022-01-26 10:21:47.858416
exception occured in name lseu-0.100000 submode remark_2 algorithm MT-K
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 18, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,False,False,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein45-0.900000 submode remark_2 algorithm MT-K
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 18, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,False,False,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein27-0.900000 submode remark_2 algorithm MT-K
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 18, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,False,False,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein45-0.500000 submode remark_2 algorithm MT-K
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 18, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,False,False,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

Run on 2022-01-26 10:23:02.039784
exception occured in name lseu-0.100000 submode remark_2 algorithm MT
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 16, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,False,False,False,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein45-0.900000 submode remark_2 algorithm MT
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 16, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,False,False,False,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein27-0.900000 submode remark_2 algorithm MT
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 16, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,False,False,False,big_M)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 71, in solve_subproblem_remark_2
    sub,y_solution = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 126, in setup_sub_rem_2
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

Run on 2022-01-26 10:24:14.089906
Run on 2022-01-26 10:24:50.380812
Run on 2022-01-26 10:27:47.591567
Run on 2022-01-26 10:38:27.713349
Run on 2022-01-26 10:38:35.981537
Run on 2022-01-26 10:43:01.279978
Run on 2022-01-26 10:43:09.825040
Run on 2022-01-26 10:54:59.414035
Run on 2022-01-26 10:55:04.074433
Run on 2022-01-26 10:55:27.516941
exception occured in name lseu-0.100000 submode regular_2 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 181, in MT
    raise ValueError('Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"')
ValueError: Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"

exception occured in name stein45-0.900000 submode regular_2 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 181, in MT
    raise ValueError('Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"')
ValueError: Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"

exception occured in name stein27-0.900000 submode regular_2 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 181, in MT
    raise ValueError('Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"')
ValueError: Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"

Run on 2022-01-26 10:55:28.033474
exception occured in name lseu-0.100000 submode regular_2 algorithm ST-K-C-S
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 30, in solve
    return ST(problem_data,tol,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 259, in ST
    raise ValueError('Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"')
ValueError: Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"

exception occured in name stein45-0.900000 submode regular_2 algorithm ST-K-C-S
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 30, in solve
    return ST(problem_data,tol,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 259, in ST
    raise ValueError('Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"')
ValueError: Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"

exception occured in name stein27-0.900000 submode regular_2 algorithm ST-K-C-S
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 30, in solve
    return ST(problem_data,tol,time_limit,subproblem_mode,True,True,True,big_M)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 259, in ST
    raise ValueError('Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"')
ValueError: Keyword argument subproblem_mode must be "regular", "remark_1" or "remark_2"

Run on 2022-01-26 10:56:03.129556
Run on 2022-01-26 10:56:08.563317
Run on 2022-01-26 11:09:03.395580
Run on 2022-01-26 11:09:08.984169
Run on 2022-01-26 14:03:02.521393
Run on 2022-01-26 14:03:08.534576
exception occured in name lseu-0.100000 submode regular algorithm ST-K-C-S
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M,True)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 30, in solve
    return ST(problem_data,tol,time_limit,subproblem_mode,True,True,True,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 269, in ST
    master = add_cut(problem_data,master,meta_data,y_var,dual_var,w_var,array(y_p))
TypeError: add_cut() missing 1 required positional argument: 'optimized_binary_expansion'

exception occured in name stein45-0.900000 submode regular algorithm ST-K-C-S
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M,True)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 30, in solve
    return ST(problem_data,tol,time_limit,subproblem_mode,True,True,True,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 269, in ST
    master = add_cut(problem_data,master,meta_data,y_var,dual_var,w_var,array(y_p))
TypeError: add_cut() missing 1 required positional argument: 'optimized_binary_expansion'

exception occured in name stein27-0.900000 submode regular algorithm ST-K-C-S
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M,True)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 30, in solve
    return ST(problem_data,tol,time_limit,subproblem_mode,True,True,True,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 269, in ST
    master = add_cut(problem_data,master,meta_data,y_var,dual_var,w_var,array(y_p))
TypeError: add_cut() missing 1 required positional argument: 'optimized_binary_expansion'

Run on 2022-01-26 14:38:26.334631
Run on 2022-01-26 14:38:31.725585
exception occured in name lseu-0.100000 submode remark_1 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M,True)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 35, in solve_subproblem_regular
    sub = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 92, in setup_sub_rem_1
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein45-0.900000 submode remark_1 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M,True)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 35, in solve_subproblem_regular
    sub = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 92, in setup_sub_rem_1
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein27-0.900000 submode remark_1 algorithm MT-K-F-W
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M,True)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 22, in solve
    return MT(problem_data,tol,iteration_limit,time_limit,subproblem_mode,True,True,True,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 35, in solve_subproblem_regular
    sub = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 92, in setup_sub_rem_1
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

Run on 2022-01-26 14:38:33.711893
Run on 2022-01-26 14:38:38.773022
Run on 2022-01-26 14:38:43.730645
exception occured in name lseu-0.100000 submode remark_1 algorithm ST-K-C-S
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M,True)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 30, in solve
    return ST(problem_data,tol,time_limit,subproblem_mode,True,True,True,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 261, in ST
    initial_solution, initial_incumbent, initial_time,initial_time_in_sub,initial_num_of_subs, initial_status,initial_gap = MT(problem_data,tol,1,time_limit,subproblem_mode,False,False,False,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 35, in solve_subproblem_regular
    sub = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 92, in setup_sub_rem_1
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein45-0.900000 submode remark_1 algorithm ST-K-C-S
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M,True)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 30, in solve
    return ST(problem_data,tol,time_limit,subproblem_mode,True,True,True,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 261, in ST
    initial_solution, initial_incumbent, initial_time,initial_time_in_sub,initial_num_of_subs, initial_status,initial_gap = MT(problem_data,tol,1,time_limit,subproblem_mode,False,False,False,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 35, in solve_subproblem_regular
    sub = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 92, in setup_sub_rem_1
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

exception occured in name stein27-0.900000 submode remark_1 algorithm ST-K-C-S
Traceback (most recent call last):
  File "/Users/felixneussel/Library/Mobile Documents/com~apple~CloudDocs/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/performance_check.py", line 253, in Test_Run_npz
    solution,obj,runtime,times_in_sub,num_of_subs, status,gap = solve(problem_data,1e-5,np.infty,TIME_LIMIT,SUBPROBLEM_MODE,ALGORITHM,BIG_M,True)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 30, in solve
    return ST(problem_data,tol,time_limit,subproblem_mode,True,True,True,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 261, in ST
    initial_solution, initial_incumbent, initial_time,initial_time_in_sub,initial_num_of_subs, initial_status,initial_gap = MT(problem_data,tol,1,time_limit,subproblem_mode,False,False,False,big_M,optimized_binary_expansion)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 207, in MT
    cut_point,solution,UB, TIME_LIMIT_EXCEEDED,time_in_sub = SOLVE_SUB_FUNCTION(SETUP_SUB_FUNCTION,UB,solution,m_vars,problem_data,master,meta_data,y_var,dual_var,w_var,cut_counter,start,time_limit)
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/multitree.py", line 35, in solve_subproblem_regular
    sub = SETUP_SUB_FUNCTION(problem_data,meta_data,getX_IParam(master))
  File "/Users/felixneussel/Documents/Uni/Vertiefung/Bachelorarbeit/Implementierung/MIQP_QP_Solver/Functional/subproblem.py", line 92, in setup_sub_rem_1
    jr,I,R,J,ll_constr,bin_coeff_dict,bin_coeff_arr = meta_data
ValueError: too many values to unpack (expected 7)

Run on 2022-01-26 14:38:45.417547
Run on 2022-01-26 14:40:43.519602
Run on 2022-01-26 14:40:48.636465
Run on 2022-01-26 14:40:53.114734
Run on 2022-01-26 14:40:58.352996
Run on 2022-01-26 14:45:06.209947
Run on 2022-01-26 14:45:12.900365
